plugin/src/models/gatsby-api.ts

```diff
@@ -83,6 +87,7 @@ export interface IPluginOptions {
     requestConcurrency?: number
   }
   excludeFieldNames?: []
+  excludeFields?: string[] // deprecated?
```

Comment:
Yeah, this is deprecated and renamed to the above `excludeFieldNames` which is an array of strings.

Reply:
i think there's a way to annotate this in jsdoc with `@deprecated` which will surface some new vscode features (autocomplete notes, rendering a strike through for deprecated methods/etc!)

Reply:
![image](https://user-images.githubusercontent.com/1368727/102259946-c2b0e700-3edd-11eb-9ca3-a3ad149285f1.png)

Reply:
Folks have been warned about this for the past 6 or 7 months https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/blob/master/plugin/src/steps/process-and-validate-plugin-options.js#L19

So I think we can remove it